examples/default.yaml

```diff
@@ -297,6 +297,7 @@ audio:
   # QEMU audiodev, e.g., "none", "coreaudio", "pa", "alsa", "oss".
   # VZ driver, use "vz" as device name
   # Choosing "none" will mute the audio output, and not play any sound.
+  # Choosing "default" will pick the default for the operating system.
```

Comment:
Can we explicitly list the default values for each os:

```yaml
# macOS: "coreaudio" or "vz", Linux: "pa", Windows: "dsound", otherwise "oss"

Reply:
Just listed all of them, like we do for the video display